modules/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"devDependencies": {
-		"@liferay/npm-scripts": "36.9.0",
+		"@liferay/npm-scripts": "36.10.0",
```

Comment:
I think this might need rebasing since we're already in that version in `master` aren't we?

Reply:
Nevermind, looks like I was on another branch and thought this had already been updated.